lib/hexpm_web/templates/package/show.html.eex

```diff
@@ -59,55 +61,74 @@ tools = [
     </div>
   <% end %>
 
-  <div class="col-md-11 with-divider no-padding">
+  <div class="col-md-11 no-padding">
+    <h3>Downloads</h3>
+    <svg viewBox="0 0 800 200" class="chart">
+      <defs>
+        <linearGradient id="grad1" gradientUnits="userSpaceOnUse" x1="0" y1="0" x2="800" y2="200">
+          <stop offset="0%" style="stop-color:#4f28a7;" />
+          <stop offset="33%" style="stop-color:#7209b7;" />
+          <stop offset="66%" style="stop-color:#b5179e;" />
+          <stop offset="100%" style="stop-color:#f72585;" />
+        </linearGradient>
+      </defs>
+
+      <%= for y <- [40, 80, 120, 160, 200] do %>
+        <line x1="0" y1="<%= y %>" x2="800" y2="<%= y %>" stroke="#ddd"></line>
+      <% end  %>
+      <polyline fill="none" stroke="url(#grad1)" stroke-width="3" stroke-linecap="round" points="<%= points %>"></polyline>
+      <polyline fill="rgba(255, 255, 255, 0.75)" points="<%= points_fill %>"></polyline>
```

Comment:
this polyline is the "shadow" if you want to remove and see how it looks.
I'd also play around with the stroke-width on the line above to see how you like it thicker and thinner.